src/devices/bus/coco/coco_fdc.cpp

```diff
@@ -2,115 +2,121 @@
 // copyright-holders:Nathan Woods
 /*********************************************************************
 
-    coco_fdc.cpp
```

Comment:
srcclean is going to change the indentation in the comment to use spaced beyond the level of the opening comment delimiter.  Please run srcclean over your modified files.

Reply:
Srcclean run. Plus other white space fixes.